scripts/orion

```diff
@@ -0,0 +1,46 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
actually, the suffix is missing as well ...
As we now are 'destroying' the broker, it cannot really be called by hand anymore, the script needs a short name, easy to use. That it invokes 3 cli by default we will just have to remember.
I would propose orion.sh.

[ But, we can always give it an unusable name and then all create aliases for it instead. I just don't see the reason for it. This script is supposed to replace the alias ]